stdlib/public/SDK/Foundation/Foundation.swift

```diff
@@ -661,15 +658,9 @@ final public class NSFastEnumerationIterator : IteratorProtocol {
   var count: Int
 
   /// Size of ObjectsBuffer, in ids.
-  var STACK_BUF_SIZE: Int { return 4 }
+  static var STACK_BUF_SIZE: Int { return 4 }
 
-  // FIXME: Replace with _CocoaFastEnumerationStackBuf.
-  /// Must have enough space for STACK_BUF_SIZE object references.
-  struct ObjectsBuffer {
-    var buf: (OpaquePointer, OpaquePointer, OpaquePointer, OpaquePointer) =
-      (nil, nil, nil, nil)
-  }
-  var objects: [ObjectsBuffer]
+  var objects: [Unmanaged<AnyObject>?]
```

Comment:
This code was using inline storage to prevent an extra allocation for the array.  Could you change it back?